invoke.example.yml

```
@@ -1,12 +1,15 @@
 ---
 nautobot_device_onboarding:
-  project_name: "nautobot-device-onboarding"
   nautobot_ver: "2.0.3"
```

Since the minimum version of nautobot is now 2.2.3 this probably shouldn't default to an incompatible version

```suggestion
  nautobot_ver: "2.2.3"
```